test/common/common/utility_speed_test.cc

```diff
@@ -0,0 +1,100 @@
+// Note: this should be run with --compilation_mode=opt.
```

Comment:
Based on a some experience doing this kind of microbenchmark work on Google workstations, I think the caveats are wider than just `opt`. For example, you want to disable cstate power management, ideally have a quiescent system and/or use CPU affinity with taskset to carve off some CPU to work on. Ideally we would run these in something equivalent to perflab.

Reply:
I added some more text to that effect, though my results were pretty stable after multiple iterations.  I guess a good place to put the results is the PR description.